tests/e2e/lifecycle/test_no_liveness_probe.py

```diff
@@ -0,0 +1,71 @@
+# BZ2142901 Automated Test
+
+from logging import getLogger
+from ocs_ci.ocs.constants import (
+    OPENSHIFT_STORAGE_NAMESPACE,
+    CSI_CEPHFSPLUGIN_LABEL,
+    POD,
+    CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+    CSI_RBDPLUGIN_LABEL,
+    CSI_RBDPLUGIN_PROVISIONER_LABEL,
+)
+from ocs_ci.ocs.ocp import OCP
+
+log = getLogger("__name__")
+
+LIVENESS_CONTAINER = "liveness-prometheus"
```

Comment:
LIVENESS_CONTAINER may be moved to ocs_ci.ocs.constants for further use (not an issue, just recommendation)

Reply:
i cant see any other use case for this const, out of this test - it should not exist on ODF